### PR TITLE
Import of TDE history

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -9234,6 +9258,7 @@ entitlied->entitled
 entitys->entities
 entoties->entities
 entoty->entity
+entquiry->enquiry
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Yikes, this is a can of worms:  
https://www.grammarly.com/blog/inquire-enquire/

```
entquire->enquire, inquire,
entquires->enquires, inquires,
etquiries->enquiries, inquiries,
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -3185,6 +3189,7 @@ bloted->bloated
 bluestooth->bluetooth
 bluetooh->bluetooth
 blured->blurred
+blurr->blur
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -6172,6 +6185,7 @@ corousel->carousel
 corparate->corporate
 corperations->corporations
 corpration->corporation
+corproration->corporation
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -6585,6 +6600,7 @@ cylnder->cylinder
 cylnders->cylinders
 cylynders->cylinders
 cymk->CMYK
+cyrlicy->cyrylicy
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: I guess it's from Czech.
Removed.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -7001,6 +7018,7 @@ defintition->definition
 defintivly->definitively
 defition->definition
 defitions->definitions
+defned->defend
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -7032,6 +7050,7 @@ degreee->degree
 degreeee->degree
 degreeees->degrees
 degreees->degrees
+degress->degrees
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -7461,6 +7482,7 @@ desparately->desperately
 despatch->dispatch
 despict->depict
 despiration->desperation
+desposit->deposit
 desposition->disposition
```
[the remark this answers is not in the document]
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

**Comment by Gelma** [comment]
I also forgot to add a second part of words from TDE.

So, I commit right now the fixes and also the rest, with a detailed commit, so you can follow changes easily,

**Pull request merged**